src/hooks/useWordlistEntriesCreate.ts

```diff
@@ -27,6 +27,7 @@ const buildOptimisticResponse = ({
         return {
           __typename: 'WordlistEntry',
           categories: categories.map(cat => ({
+            createdAt: cat.createdAt,
```

Comment:
category doesn't have an id but has a `createdAt` property?

Reply:
this should be an arbitrary string because the category may or may not be a new one

Reply:
this property is set elsewhere and is an arbitrary string, which is correct